src/Symfony/Component/HttpClient/Response/NativeResponse.php

```diff
@@ -162,7 +166,7 @@ private function open(): void
     /**
      * {@inheritdoc}
      */
-    private function close(): void
+    protected function close(): void
```

Comment:
private was good - traits are not contracts (same below)

Reply:
This was done inconsistently. Sometimes the overridden trait methods were private, sometimes protected.

Apart from that, PhpStorm has troubles inspecting this, but this is probably something JetBrains should fix.